website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -146,12 +148,60 @@ The minimum custom permissions set for a workspace is the permission to read run
 
 Some permissions - such as the runs permission - are tiered: you can assign one permission per category, since higher permissions include all of the capabilities of the lower ones.
 
+## Project Permissions
+
+You can assign project-specific permissions to teams.
+
+### Fixed Permission Sets
+
+Fixed permission sets are bundles of specific permissions for a project, designed for common usage patterns. You can only assign project-specific permissions through one of these permission sets.
+
+#### Project Admins
```

Comment:
```suggestion
#### Project Admin
```

Reply:
Changes made in this PR to be merged upon approval: https://github.com/hashicorp/terraform-docs-common/pull/245